packages/hardware-trezor/src/TrezorKeyAgent.ts

```diff
@@ -163,12 +163,9 @@ export class TrezorKeyAgent extends KeyAgentBase {
     const scope = new ManagedFreeableScope();
     try {
       await this.isTrezorInitialized;
-      const cslTxBody = coreToCml.txBody(scope, tx.body);
       const trezorTxData = await txToTrezor({
-        accountIndex: this.accountIndex,
         cardanoTxBody: tx.body,
         chainId: this.chainId,
-        cslTxBody,
         inputResolver: this.inputResolver,
         knownAddresses: this.knownAddresses
       });
```

Comment:
I replicated this from https://github.com/input-output-hk/cardano-js-sdk/pull/903 to avoid additional work once both PR's are merged